### PR TITLE
Split Into FeedWheel and revolver

### LINKED ISSUE #55: Split revolver into feedWheel and revolver

### PR DESCRIPTION
**Pull request by Jacob4649**
Fix #55

### CODE REVIEW

**Comment by Jacob4649** [comment]
Merge #56 first

File: src/main/java/frc/robot/commands/revolver/RunRevolver.java [hunk 1]
```diff
@@ -12,16 +12,15 @@
 
 public class RunRevolver extends CommandBase {
 
-  private double m_revolverSpeed, m_wheelSpeed;
+  private double m_revolverSpeed;
```
Comment (matthew-reynolds) [inline src/main/java/frc/robot/commands/revolver/RunRevolver.java]: Nitpick: Can simplify to `m_speed`

**Review by matthew-reynolds — APPROVED** [review]

File: src/main/java/frc/robot/commands/revolver/RunRevolver.java (hunk 1 above)
Comment (Jacob4649) [reply src/main/java/frc/robot/commands/revolver/RunRevolver.java]: Resolved in d9c4787

**Comment by matthew-reynolds** [comment]
Approved, waiting until #56 is merged.

**Review by matthew-reynolds — APPROVED** [review]

**Pull request merged**